.github/workflows/build-test-distribute-flavor.yml

```
@@ -309,6 +337,124 @@ jobs:
+  # opts in (typically vllm). Gated on cuda_version == '12.9' so it only runs
+  # once when the caller matrixes over multiple cuda versions.
+  # ============================================================================
+
+  sample-test:
+    if: |
+        inputs.run_sample_tests &&
+        inputs.build_image &&
+        inputs.cuda_version == '12.9'
+    needs: [build]
+    name: Sample test ${{ inputs.cpu_only && 'cpu' || format('cuda{0}', inputs.cuda_version) }}
+    runs-on: prod-tester-amd-gpu-v1 # TODO: CPU-only DinD runner when available
```

actionlint (1.7.12)

[error] label "prod-tester-amd-gpu-v1" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)